packages/make-styles/src/runtime/utils/hashSequence.ts

```diff
@@ -1,6 +1,20 @@
-import { SEQUENCE_PREFIX } from '../../constants';
-import { hashString } from './hashString';
+import hash from '@emotion/hash';
```

Comment:
new OSS dependency ? license scan ?

Reply:
Nope, we already have it, see deleted `hashString.ts`.